src/constants.tsx

```diff
@@ -265,68 +265,17 @@ export const ADD = 'Add';
 export type ADD = typeof ADD;
```

Comment:
@cKellyDesign I think we should move "display" strings to something like `lang.ts` so as to completely separate them from other constants that should not be translated.